debian/control

```diff
@@ -0,0 +1,33 @@
+Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+Build-Depends: debhelper-compat (= 13),
+               dh-apport,
+               dh-cargo,
+               dh-golang,
+               dbus,
+               golang-go (>= 2:1.21~),
+               libpam0g-dev,
+               libc6-dev (>= 2.35),
+               pkg-config,
+               protobuf-compiler,
+Standards-Version: 4.6.2
+XS-Go-Import-Path: github.com/ubuntu/authd
+XS-Vendored-Sources-Rust: addr2line@0.21.0, adler@1.0.2, aho-corasick@1.0.5, anyhow@1.0.75, async-stream-impl@0.3.5, async-stream@0.3.5, async-trait@0.1.73, autocfg@1.1.0, axum-core@0.3.4, axum@0.6.20, backtrace@0.3.69, base64@0.21.3, bitflags@1.3.2, bitflags@2.4.0, bytes@1.5.0, cc@1.0.84, cfg-if@1.0.0, colored@2.0.4, ctor@0.2.5, deranged@0.3.8, either@1.9.0, equivalent@1.0.1, errno-dragonfly@0.1.2, errno@0.3.3, error-chain@0.12.4, fastrand@2.0.0, fixedbitset@0.4.2, fnv@1.0.7, futures-channel@0.3.28, futures-core@0.3.28, futures-sink@0.3.28, futures-task@0.3.28, futures-util@0.3.28, getrandom@0.2.10, gimli@0.28.0, h2@0.3.21, hashbrown@0.12.3, hashbrown@0.14.0, heck@0.4.1, hermit-abi@0.3.2, home@0.5.5, hostname@0.3.1, http-body@0.4.5, http@0.2.9, httparse@1.8.0, httpdate@1.0.3, hyper-timeout@0.4.1, hyper@0.14.27, indexmap@1.9.3, indexmap@2.0.0, is-terminal@0.4.9, itertools@0.11.0, itoa@1.0.9, lazy_static@1.4.0, libc@0.2.150, libnss@0.5.0, linux-raw-sys@0.4.5, log@0.4.20, match_cfg@0.1.0, matchit@0.7.2, memchr@2.6.3, mime@0.3.17, miniz_oxide@0.7.1, mio@0.8.9, multimap@0.8.3, num_cpus@1.16.0, num_threads@0.1.6, object@0.32.1, once_cell@1.18.0, paste@1.0.14, percent-encoding@2.3.0, petgraph@0.6.4, pin-project-internal@1.1.3, pin-project-lite@0.2.13, pin-project@1.1.3, pin-utils@0.1.0, ppv-lite86@0.2.17, prettyplease@0.2.14, proc-macro2@1.0.66, prost-build@0.12.0, prost-derive@0.12.3, prost-types@0.12.0, prost@0.12.3, quote@1.0.33, rand@0.8.5, rand_chacha@0.3.1, rand_core@0.6.4, redox_syscall@0.3.5, regex-automata@0.3.8, regex-syntax@0.7.5, regex@1.9.5, rustc-demangle@0.1.23, rustix@0.38.11, rustversion@1.0.14, serde@1.0.188, serde_derive@1.0.188, simple_logger@4.3.0, slab@0.4.9, socket2@0.4.9, socket2@0.5.5, syn@2.0.31, sync_wrapper@0.1.2, syslog@6.1.0, tempfile@3.8.0, time-core@0.1.1, time-macros@0.2.14, time@0.3.28, tokio-io-timeout@1.2.0, tokio-macros@2.2.0, tokio-stream@0.1.14, tokio-util@0.7.8, tokio@1.34.0, tonic-build@0.10.2, tonic@0.10.2, tower-layer@0.3.2, tower-service@0.3.2, tower@0.4.13, tracing-attributes@0.1.26, tracing-core@0.1.31, tracing@0.1.37, try-lock@0.2.4, unicode-ident@1.0.11, version_check@0.9.4, want@0.3.1, wasi@0.11.0+wasi-snapshot-preview1, which@4.4.2, winapi-i686-pc-windows-gnu@0.4.0, winapi-x86_64-pc-windows-gnu@0.4.0, winapi@0.3.9, windows-sys@0.48.0, windows-targets@0.48.5, windows_aarch64_gnullvm@0.48.5, windows_aarch64_msvc@0.48.5, windows_i686_gnu@0.48.5, windows_i686_msvc@0.48.5, windows_x86_64_gnu@0.48.5, windows_x86_64_gnullvm@0.48.5, windows_x86_64_msvc@0.48.5
```

Comment:
Please use multi-line or it will be a mess to track changes.

```suggestion
XS-Vendored-Sources-Rust: addr2line@0.21.0,
                          adler@1.0.2,
                          aho-corasick@1.0.5,
                          anyhow@1.0.75,
                          async-stream-impl@0.3.5,
                          async-stream@0.3.5,
                          async-trait@0.1.73,
                          autocfg@1.1.0,
                          axum-core@0.3.4,
                          axum@0.6.20,
                          backtrace@0.3.69,
                          base64@0.21.3,
                          bitflags@1.3.2,
                          bitflags@2.4.0,
                          bytes@1.5.0,
                          cc@1.0.84,
                          cfg-if@1.0.0,
                          colored@2.0.4,
                          ctor@0.2.5,
                          deranged@0.3.8,
                          either@1.9.0,
                          equivalent@1.0.1,
                          errno-dragonfly@0.1.2,
                          errno@0.3.3,
                          error-chain@0.12.4,
                          fastrand@2.0.0,
                          fixedbitset@0.4.2,
                          fnv@1.0.7,
                          futures-channel@0.3.28,
                          futures-core@0.3.28,
                          futures-sink@0.3.28,
                          futures-task@0.3.28,
                          futures-util@0.3.28,
                          getrandom@0.2.10,
                          gimli@0.28.0,
                          h2@0.3.21,
                          hashbrown@0.12.3,
                          hashbrown@0.14.0,
                          heck@0.4.1,
                          hermit-abi@0.3.2,
                          home@0.5.5,
                          hostname@0.3.1,
                          http-body@0.4.5,
                          http@0.2.9,
                          httparse@1.8.0,
                          httpdate@1.0.3,
                          hyper-timeout@0.4.1,
                          hyper@0.14.27,
                          indexmap@1.9.3,
                          indexmap@2.0.0,
                          is-terminal@0.4.9,
                          itertools@0.11.0,
                          itoa@1.0.9,
                          lazy_static@1.4.0,
                          libc@0.2.150,
                          libnss@0.5.0,
                          linux-raw-sys@0.4.5,
                          log@0.4.20,
                          match_cfg@0.1.0,
                          matchit@0.7.2,
                          memchr@2.6.3,
                          mime@0.3.17,
                          miniz_oxide@0.7.1,
                          mio@0.8.9,
                          multimap@0.8.3,
                          num_cpus@1.16.0,
                          num_threads@0.1.6,
                          object@0.32.1,
                          once_cell@1.18.0,
                          paste@1.0.14,
                          percent-encoding@2.3.0,
                          petgraph@0.6.4,
                          pin-project-internal@1.1.3,
                          pin-project-lite@0.2.13,
                          pin-project@1.1.3,
                          pin-utils@0.1.0,
                          ppv-lite86@0.2.17,
                          prettyplease@0.2.14,
                          proc-macro2@1.0.66,
                          prost-build@0.12.0,
                          prost-derive@0.12.3,
                          prost-types@0.12.0,
                          prost@0.12.3,
                          quote@1.0.33,
                          rand@0.8.5,
                          rand_chacha@0.3.1,
                          rand_core@0.6.4,
                          redox_syscall@0.3.5,
                          regex-automata@0.3.8,
                          regex-syntax@0.7.5,
                          regex@1.9.5,
                          rustc-demangle@0.1.23,
                          rustix@0.38.11,
                          rustversion@1.0.14,
                          serde@1.0.188,
                          serde_derive@1.0.188,
                          simple_logger@4.3.0,
                          slab@0.4.9,
                          socket2@0.4.9,
                          socket2@0.5.5,
                          syn@2.0.31,
                          sync_wrapper@0.1.2,
                          syslog@6.1.0,
                          tempfile@3.8.0,
                          time-core@0.1.1,
                          time-macros@0.2.14,
                          time@0.3.28,
                          tokio-io-timeout@1.2.0,
                          tokio-macros@2.2.0,
                          tokio-stream@0.1.14,
                          tokio-util@0.7.8,
                          tokio@1.34.0,
                          tonic-build@0.10.2,
                          tonic@0.10.2,
                          tower-layer@0.3.2,
                          tower-service@0.3.2,
                          tower@0.4.13,
                          tracing-attributes@0.1.26,
                          tracing-core@0.1.31,
                          tracing@0.1.37,
                          try-lock@0.2.4,
                          unicode-ident@1.0.11,
                          version_check@0.9.4,
                          want@0.3.1,
                          wasi@0.11.0+wasi-snapshot-preview1,
                          which@4.4.2,
                          winapi-i686-pc-windows-gnu@0.4.0,
                          winapi-x86_64-pc-windows-gnu@0.4.0,
                          winapi@0.3.9,
                          windows-sys@0.48.0,
                          windows-targets@0.48.5,
                          windows_aarch64_gnullvm@0.48.5,
                          windows_aarch64_msvc@0.48.5,
                          windows_i686_gnu@0.48.5,
                          windows_i686_msvc@0.48.5,
                          windows_x86_64_gnu@0.48.5,
                          windows_x86_64_gnullvm@0.48.5,
                          windows_x86_64_msvc@0.48.5,
```

Reply:
This must be a *single line*, as expected by `dh-cargo-vendored-sources`.

Reply:
`dh-cargo-vendored-sources` as every debian tool should definitely be able to parse things multiline.

However, since it's generated it seems that making it multi-line is more problematic.